packages/edit-site/src/components/page-templates/index.js

```diff
@@ -38,44 +38,26 @@ const EMPTY_ARRAY = [];
 
 const defaultLayouts = {
 	[ LAYOUT_TABLE ]: {
-		fields: [ 'template', 'author' ],
+		showMedia: false,
```

Comment:
This is a naming nit: the special fields are visible by default if they are declared by the layout. So, what we want is to **hide** them in certain scenarios (table & list layout, in this case). I'd go with `hideMedia` and `hideDescription` instead.

Not a enormous problem, but it's confusing if you declare `showMedia: true` but don't declare any media field. It may lead you to believe that a placeholder is rendered — which is not true. The opposite doesn't create the same confusion. A nit, anyways.

Reply:
I hesitated to be honest. Ideally I think the default value shouldn't have to impact the naming of these properties. I felt that "showSomething: true/false" is actually easier to understand than "hideSomething: true/false" but I can update. WDYT

Reply:
I think I'd like to revisit this when removing the `enableHiding` from the Fields API. Perhaps something that controls the toggeability and visibility of a field can be grouped together. If it was to stay as it is, I'd prefer `hide` over `show` as prefix in this case for the reasons I shared — but not a big deal, both can probably work fine.